packages/playground/blueprints/src/lib/steps/define-wp-config-consts.ts

```diff
@@ -8,42 +8,46 @@ export interface DefineWpConfigConstsStep {
 	step: 'defineWpConfigConsts';
 	/** The constants to define */
 	consts: Record<string, unknown>;
+	/** Whether to virtualize the wp-config.php and playground-consts.json files, by default false */
```

Comment:
Let’s explain what virtualization means here and how it won’t affect the local project files

Reply:
done! https://github.com/WordPress/wordpress-playground/pull/415/commits/ad89004cdcce4a5395cb209567c19193489977c1